CMakeLists.txt

```diff
@@ -243,10 +243,12 @@ if (${BUILD_CLI})
   add_subdirectory(plugins/SQLiteCLI)
 endif()
 
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/quickstart/ DESTINATION SampleCode/quickstart USE_SOURCE_PERMISSIONS)
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/cppcache/integration-test/keystore/ DESTINATION SampleCode/quickstart/keystore)
-install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/xsds/ DESTINATION xsds)
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/examples/ DESTINATION examples)
+#install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/cppcache/integration-test/keystore/ DESTINATION SampleCode/quickstart/keystore)
+#install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/xsds/ DESTINATION xsds)
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/defaultSystem/ DESTINATION defaultSystem)
+#install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/cppcache/include/geode/internal DESTINATION include/geode/)
+#install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/cppcache/include/geode/utils DESTINATION include/geode/)
```

Comment:
What's with the rest of these commented out sections?

Reply:
Delete, don't comment.

Reply:
These two lines are cruft and should be pulled.